person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def iinitialize(name, age = 'unknown', parent_permission = 'true')
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # getter
+  attr_accessor :name
+  attr_accessor :age
+  attr_reader :parent_permission
```

Comment:
- [ ] Is the id that should only be able to be retrieved so only the Getter is defined and not `parent_permission` 🙃🙃